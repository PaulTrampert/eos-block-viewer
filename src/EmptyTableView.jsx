import React from 'react';
import styles from './EmptyTableView.module.css';

const EmptyTableView = () => {
  return (
    <tr>
      <td colSpan={3} className={styles.empty}>
        Nothing to show. Click 'Load' to get started.
      </td>
    </tr>
  )
}

export default EmptyTableView;