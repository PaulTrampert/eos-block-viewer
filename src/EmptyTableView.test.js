import React from 'react';
import {shallow} from 'enzyme';
import EmptyTableView from './EmptyTableView';

describe('EmptyTableView', () => {
  let subject;

  beforeEach(() => {
    subject = shallow(<EmptyTableView />);
  });

  it('Renders the message to click "Load" to get started.', () => {
    expect(subject).toMatchSnapshot();
  });
});