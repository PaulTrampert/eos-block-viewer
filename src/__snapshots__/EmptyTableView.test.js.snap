// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyTableView Renders the message to click "Load" to get started. 1`] = `
<tr>
  <td
    className="empty"
    colSpan={3}
  >
    Nothing to show. Click 'Load' to get started.
  </td>
</tr>
`;
